### PR TITLE
WIP: Use queue manager to kill jobs

### PR DESCRIPTION
**Pull request by ManInFez**
**Issue**
Resolves #my_issue

**Approach**
_Short description of the approach_

### CODE REVIEW

File: python/res/job_queue/job_queue_manager.py [hunk 1]
```diff
@@ -155,6 +155,15 @@ def execute_queue(self):
...
+                for job in job_queue.job_list:
+                    job.stop(job_queue.driver)
 
         for thread in started_job_threads:
             thread.join()
+
+        #Cleanup all not-started jobs
+        for job in job_queue.job_list:
+            if job.status == JobStatusType.JOB_QUEUE_WAITING:
+                job._set_status(JobStatusType.JOB_QUEUE_IS_KILLED)
```
Comment (andreabrambilla) [inline python/res/job_queue/job_queue_manager.py:168]: This looks a bit like a hack. I would guess that `job.stop` handles the status change for a job that is running. I don't see why the jobs that are waiting should be treated differently.
If this is really necessary, the comment should be more exhaustive

**Comment by markusdregi** [comment]
This PR has neither an issue nor a description. Please add some more information....

**Comment by xjules** [comment]
> This PR has neither an issue nor a description. Please add some more information....

This PR is now included in #708, can go away after #708 is merged.

File: python/res/job_queue/job_queue_manager.py (hunk 1 above)
Comment (xjules) [reply python/res/job_queue/job_queue_manager.py:168]: > This looks a bit like a hack. I would guess that `job.stop` handles the status change for a job that is running. I don't see why the jobs that are waiting should be treated differently.
> If this is really necessary, the comment should be more exhaustive

This's to get the correct status on non started jobs (after being stopped by the user. Agree, will include a comment there.

**Pull request closed without merging**